include/swift/ABI/MetadataValues.h

```diff
@@ -47,6 +47,17 @@ enum {
   /// in a default actor.
   NumWords_DefaultActor = 10,
 
+  /// The number of words (in addition to heap object header)
+  /// in a distributed *remote* (proxy) actor.
+  ///
+  /// It must be greater than:
+  /// NumWords =
+  ///   NumWords_DefaultActor
+  ///   + (sizeof(ActorTransport) == 88)
+  ///   + transport pointer
+  ///   + $impl pointer
+  NumWords_DistributedRemoteActor = 10 + 11 + 2 + 2 + 4/*reserved*/,
```

Comment:
This can go away completely since we moved to the enum representation rather than custom remote instance allocation; for now at least...